fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -35,53 +54,50 @@ def __init__(self, X, A, y, learner, constraints, eps, B, opt_lambda=True):
         self.last_linprog_n_hs = 0
         self.last_linprog_result = None
 
-    def eval_from_error_gamma(self, error, gamma, lambda_vec):
+    def _eval_from_error_gamma(self, error, gamma, lambda_vec):
         """ Return the value of the Lagrangian.
-        Returned values:
-        L -- value of the Lagrangian
-        L_high -- value of the Lagrangian under the best
-        response of the lambda player
+
+        :return: tuple `(L, L_high)` where `L` is the value of the Lagrangian, and
+            `L_high` is the value of the Lagrangian under the best response of the lambda player
+        :rtype: tuple of two floats
         """
         lambda_projected = self.constraints.project_lambda(lambda_vec)
         if self.opt_lambda:
-            L = error + np.sum(lambda_projected * gamma) \
-                - self.eps * np.sum(lambda_projected)
+            L = error + np.sum(lambda_projected * gamma) - self.eps * np.sum(lambda_projected)
```

Comment:
I take it we're allowing more text per line that when this originally formatted?

Reply:
Right, and equations like this should never go across lines